test/blackbox-tests/test-cases/action-modifying-a-dependency/run.t

```diff
@@ -0,0 +1,21 @@
+In this test the "x" alias depends on the file "data" but the action
+associated to "x" appends a line to "data". The current behavior is
```

Comment:
It's good to have a neutral description of the behavior, but do you think that's the expected behavior? Is it worth adding a line on what this "should" do instead?

Reply:
Indeed, I added a description of what it should do.